dtests/src/test/java/io/snappydata/hydra/cluster/SnappyPrms.java

```diff
@@ -30,6 +30,12 @@
    */
   public static Long sqlScriptNames;
 
+  /**
+   * Parameter used to get the user specified script names.
+   * (VectosetValues of Strings) A list of values for script Names to execute.
+   */
+  public static Long scriptNames;
+
```

Comment:
can we reuse sqlScriptNames here?

Reply:
These are not SQL scripts. The parameter is mainly to pass the shell script names for execution.